IaC/ps57.cd/init.groovy.d/cloud.groovy

```diff
@@ -180,13 +180,31 @@ initMap['micro-amazon'] = '''
             sudo mount ${DEVICE} /mnt
         fi
     fi
+    SYSREL=$(cat /etc/system-release | tr -dc '0-9.'|awk -F'.' {'print $1'})
     until sudo yum makecache; do
         sleep 1
         echo try again
     done
-    sudo yum -y install java-1.8.0-openjdk git || :
-    sudo yum -y install aws-cli || :
-    sudo yum -y remove java-1.7.0-openjdk || :
+    if [[ $SYSREL -eq 2 ]]; then
+        sudo amazon-linux-extras install epel -y
+    else
+        sudo yum -y install epel-release
+    fi
+    sudo yum -y install java-1.8.0-openjdk git p7zip || :
+    sudo yum -y remove java-1.7.0-openjdk aws-cli || :
+
+    if ! $(aws --version | grep -q 'aws-cli/2'); then
+        find /tmp -maxdepth 1 -name "*aws*" | xargs sudo rm -rf
+
+        until curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2.zip"; do
+            sleep 1
+            echo try again
+        done
+
+        7za -o/tmp x /tmp/awscliv2.zip 
```

Comment:
it will fail if aws dir is available

Reply:
It won't: https://github.com/Percona-Lab/jenkins-pipelines/pull/917/files#diff-b6ffba51adae65104db98666c6d5446fff1b01cfc9e891d410d67f13aa332e4cR197